File: src/main/java/com/company/payroll/mapper/TitleMapper.java
package com.company.payroll.mapper;

import org.springframework.stereotype.Repository;

import com.company.payroll.model.Title;

@Repository
public interface TitleMapper {
    int deleteByPrimaryKey(Integer titleno);

    int insert(Title row);

    int insertSelective(Title row);

    Title selectByPrimaryKey(Integer titleno);

    int updateByPrimaryKeySelective(Title row);

    int updateByPrimaryKey(Title row);
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/mapper/TitleMapper.kt
package com.company.payroll.mapper

import com.company.payroll.model.Title
import org.springframework.stereotype.Repository

@Repository
interface TitleMapper {
  fun deleteByPrimaryKey(titleNo: Int): Int

  fun insert(row: Title): Int

  fun insertSelective(row: Title): Int

  fun selectByPrimaryKey(titleNo: Int): Title?

  fun updateByPrimaryKeySelective(row: Title): Int

  fun updateByPrimaryKey(row: Title): Int
}